skate-plugin/build.gradle.kts

```diff
@@ -1,8 +1,9 @@
+
 import com.jetbrains.plugin.structure.base.utils.exists
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import java.nio.file.Paths
-import java.util.Locale
+import java.util.*
```

Comment:
you can fix your IDE settings to avoid this

Settings > Editor > Code Style > Kotlin > Imports

Set everything to 99 and remove package imports

![Uploading image.png…]()